x11-wm/e16/e16-1.0.30.ebuild

```diff
@@ -0,0 +1,146 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Enlightenment Window Manager (E16)"
+HOMEPAGE="https://www.enlightenment.org https://sourceforge.net/projects/enlightenment/"
+SRC_URI="https://downloads.sourceforge.net/projects/enlightenment/${P}.tar.xz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="amd64 x86"
+IUSE="+alsa audiofile dbus debug +dialogs doc examples gnome
+libhack modules nls no-container opengl +pango +poll player
+pulseaudio select +sndfile sound +themes xcomposite +xft xi2
+xinerama xpresent +xrandr +xrender +xsm +xsync zoom"
+
+REQUIRED_USE="
+	^^ ( poll select )
+	opengl? ( xcomposite )
+	pango? ( xft )
+	sound? (
+		^^ ( alsa player pulseaudio )
+		alsa?       ( ^^ ( sndfile audiofile ) )
+		pulseaudio? ( ^^ ( sndfile audiofile ) )
+	)
+"
+
+BDEPEND="
+	nls? ( sys-devel/gettext )
+	virtual/pkgconfig
+"
+COMMON_DEPEND="
+	media-libs/freetype:2
+	media-libs/imlib2[X]
+	virtual/libiconv
+	x11-libs/libX11
+	x11-libs/libXext
+	x11-misc/xbitmaps
+	dbus? ( sys-apps/dbus )
+	opengl? (
+		media-libs/glu
+		media-libs/mesa
+	)
+	pango? (
+		dev-libs/glib:2
+		x11-libs/pango[X]
+	)
+	sound? (
+		alsa? ( media-libs/alsa-lib )
+		player? ( media-sound/alsa-utils )
+		pulseaudio? ( || (
+			media-libs/libpulse
+			media-sound/apulse[sdk]
+		) )
+		audiofile? ( media-libs/audiofile:= )
+		sndfile? ( media-libs/libsndfile )
+	)
+	xcomposite? (
+		x11-libs/libXcomposite
+		x11-libs/libXdamage
+		x11-libs/libXfixes
+	)
+	xft? ( x11-libs/libXft )
+	xinerama? ( x11-libs/libXinerama )
+	xpresent? ( x11-libs/libXpresent )
+	xrandr? ( x11-libs/libXrandr )
+	xrender? ( x11-libs/libXrender )
+	xsm? (
+		x11-libs/libICE
+		x11-libs/libSM
+	)
+	zoom? ( x11-libs/libXxf86vm )
+"
+RDEPEND="${COMMON_DEPEND}
+	doc? ( app-doc/e16-docs )
+	nls? ( virtual/libintl )
+	themes? ( x11-themes/e16-themes )
+	!x11-wm/enlightenment:0
+"
```

Comment:
`qa-vdb` suggests more RDEPENDS are needed: `media-libs/fontconfig`, `media-libs/harfbuzz:=` and `media-libs/libglvnd[X]` (probably caused by USE=opengl).

Also, please check if `media-libs/mesa` and `x11-libs/libXxf86vm` are really needed.